### PR TITLE
fix: preserving v1alpha1 enable node ports false value avoiding CRD d…

### LINKED ISSUE #342: Missing migration of enable-node-ports annotation from v1alpha1 to v1beta1
# Bug description

Missing migration of enable-node-ports annotation from v1alpha1 to v1beta1

# How to reproduce

```yaml
kubectl create -f - << EOF
apiVersion: capsule.clastix.io/v1alpha1
kind: Tenant
metadata:
  name: oil
  annotations:
    capsule.clastix.io/enable-node-ports: "false"
spec:
  owner: # required
    name: alice
    kind: User
EOF
```

after migration becomes

```yaml
apiVersion: capsule.clastix.io/v1beta1
kind: Tenant
metadata:
  name: oil
spec:
  owners:
  - kind: User
    name: alice
  enableNodePorts: true
```
# Expected behavior

```yaml
apiVersion: capsule.clastix.io/v1beta1
kind: Tenant
metadata:
  name: oil
spec:
  owners:
  - kind: User
    name: alice
  enableNodePorts: false

```

# Logs

If applicable, please provide logs of `capsule`.

In a standard stand-alone installation of Capsule,
you'd get this by running `kubectl -n capsule-system logs deploy/capsule-controller-manager`.

# Additional context

- Capsule version: `v0.1.0-rc3 98e1640.dev`
- Helm Chart version: (`helm list -n capsule-system`)
- Kubernetes version: (`kubectl version`)

### PR DESCRIPTION
**Pull request by prometherion**
Closes #342.

### CODE REVIEW

**Comment by prometherion** [comment]
@MaxFedotov I'd like to get a review from you too.

Although the conversion webhook test was ok it seems OAPIv3 is doing some trickery stuff, so I had to use a pointer for the `enableNodePorts` value. It seems doing the trick.

```
$: kubectl create -f - << EOF
apiVersion: capsule.clastix.io/v1alpha1
kind: Tenant
metadata:
  name: oil
  annotations:
    capsule.clastix.io/enable-node-ports: "false"
spec:
  owner:
    name: alice
    kind: User
EOF

$: kubectl get tenants.v1beta1.capsule.clastix.io oil -o yaml
apiVersion: capsule.clastix.io/v1beta1
kind: Tenant
metadata:
  creationTimestamp: "2021-07-17T10:24:52Z"
  generation: 1
  name: oil
  resourceVersion: "5310"
  uid: ba1f3b2d-a204-4dfc-9188-915bca77a0f4
spec:
  enableNodePorts: false
  owners:
  - kind: User
    name: alice
status:
  size: 0
  state: active
```

tl;dr; the unbearable fallacies of unit testing 🤓

**Review by bsctl — APPROVED** [review]
lgtm @prometherion it works now
